test/adapter/json_api/transform_test.rb

```diff
@@ -102,8 +102,8 @@ def test_success_document_transform_global_config
           mock_request
           result = with_config(key_transform: :camel_lower) do
             serializer = PostSerializer.new(@post)
-            adapter = ActiveModelSerializers::Adapter::JsonApi.new(serializer)
-            adapter.serializable_hash(@options)
+            adapter = ActiveModelSerializers::Adapter::JsonApi.new(serializer, @options)
```

Comment:
So, I added `gem 'minitest-bisect'` to my Gemfile.local and ran

```bash
RUBYOPT=-Ilib:test MTB_VERBOSE=2 be minitest_bisect --seed 48816  $(find test -type f -name \*_test.rb)
```

and minitest-bisect tells me

```plain
Final reproduction:

Run options: --seed 48816 -n "/^(?:ActiveModel::Serializer::OptionsTest#(?:test_no_option_is_passed_in)|ActionController::Serialization::JsonApi::KeyTransformTest#(?:test_render_resource_with_transform))$/"

# Running:

.F

Finished in 0.039316s, 50.8697 runs/s, 50.8697 assertions/s.

  1) Failure:
ActionController::Serialization::JsonApi::KeyTransformTest#test_render_resource_with_transform [/Users/benjamin/files/dev/repos/active_model_serializers/test/action_controller/json_api/transform_test.rb:105]:
--- expected
+++ actual
@@ -1 +1 @@
-{"Data"=>{"Id"=>"1337", "Type"=>"Posts", "Attributes"=>{"Title"=>"Title 1", "Body"=>"Body 1", "PublishAt"=>"2020-03-16T03:55:25.291Z"}, "Relationships"=>{"Author"=>{"Data"=>{"Id"=>"1", "Type"=>"Authors"}}, "TopComments"=>{"Data"=>[{"Id"=>"7", "Type"=>"TopComments"}, {"Id"=>"12", "Type"=>"TopComments"}]}}, "Links"=>{"PostAuthors"=>"https://example.com/post_authors"}, "Meta"=>{"Rating"=>5, "FavoriteCount"=>10}}}
+{"data"=>{"id"=>"1337", "type"=>"posts", "attributes"=>{"title"=>"Title 1", "body"=>"Body 1", "publish-at"=>"2020-03-16T03:55:25.291Z"}, "relationships"=>{"author"=>{"data"=>{"id"=>"1", "type"=>"authors"}}, "top-comments"=>{"data"=>[{"id"=>"7", "type"=>"top-comments"}, {"id"=>"12", "type"=>"top-comments"}]}}, "links"=>{"post-authors"=>"https://example.com/post_authors"}, "meta"=>{"rating"=>5, "favorite-count"=>10}}}
```